### PR TITLE
Docs: Update quickstart instructions

### PR DESCRIPTION
**Pull request by kfaraz**
We need not specify a Druid version in the quickstart, because the previous step instructs downloading the latest version anyway.

### CODE REVIEW

**Review by techdocsmith — APPROVED** [review]
LGTM

**Review by vogievetsky — APPROVED** [review]

File: docs/tutorials/index.md [hunk 1]
```diff
@@ -83,7 +81,7 @@ From the apache-druid-{{DRUIDVERSION}} package root, run the following command:
 ./bin/start-druid
 ```
 
-This brings up instances of ZooKeeper and the Druid services:
+This brings up instances of ZooKeeper and the Druid services and may use up to 80% of the total available system memory. You may also explicitly specify the total memory for Druid to use by passing a value for the memory parameter, e.g. `./bin/start-druid -m 16g` or `./bin/start-druid --memory 16g`.
```
Comment (techdocsmith) [inline docs/tutorials/index.md:84]:
```suggestion
This brings up instances of ZooKeeper and the Druid services and may use up to 80% of the total available system memory. To explicitly set the total memory available to Druid, pass a value for the memory parameter, e.g. `./bin/start-druid -m 16g` or `./bin/start-druid --memory 16g`.
```

**Review by techdocsmith — APPROVED** [review]
small suggestion otherwise LGTM

**Comment by kfaraz** [comment]
Thanks for the reviews, @techdocsmith , @vogievetsky !
Merging since all the static checks have passed and there are no code changes in this PR.

**Pull request merged**